bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -38,6 +38,12 @@
   <div id="try-hello">
     <div class="container">
       <header role="banner">
+        {% if switch('firefox-hello-eol-notification') %}
```

Comment:
Talked to @flodolo on IRC and we're not going to add an l10n flag for this string, as it would be better to show the warning in English than not to show it at all.